sdk/connectedvmware/Azure.ResourceManager.ConnectedVmware/src/autorest.md

```diff
@@ -0,0 +1,29 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+namespace: Azure.ResourceManager.ConnectedVMware
+require: https://github.com/Azure/azure-rest-api-specs/blob/58891380ba22c3565ca884dee3831445f638b545/specification/connectedvmware/resource-manager/readme.md
+clear-output-folder: true
+output-folder: Generated/
+mgmt-debug:
+  suppress-list-exception: true
```

Comment:
We should not add this flag, instead we need to explicitly suppress the list-exceptions if there is any.
Also I suggest we could add the `show-request-path` flag here
```suggestion
  show-request-path: true
```